doc/src/manual/functions.md

```diff
@@ -143,7 +143,7 @@ julia> +(1,2,3)
 
 The infix form is exactly equivalent to the function application form -- in fact the former is
 parsed to produce the function call internally. This also means that you can assign and pass around
-operators such as [`+`](@ref) and [`*`](@ref) just like you would with other function values:
+operators such as [`+()`](@ref) and [`*()`](@ref) just like you would with other function values:
```

Comment:
Looks like this was wrong on `stable`, but the intent here is to link to `+`, not call it